src/utils/hints/accountHints.js

```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Current amount of AE owned by the account',
```

Comment:
How about "Amount of AE currently owned by the account"

Reply:
Nice suggestion, accepted